playbooks/operationalizing/external-jenkins-integration.adoc

```diff
@@ -0,0 +1,339 @@
+
+[options="header"]
+|=========================================================
+|Name |Location
+
+|base| *registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7*
+
+|maven| *registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7* 
+
+|nodejs| *registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7*
+
+|=========================================================
+
+Additional information on the Jenkins slaves provided by OpenShift can be found link:https://docs.openshift.com/container-platform/latest/using_images/other_images/jenkins.html#using-the-jenkins-kubernetes-plug-in-to-run-jobs[here].
+
+To create a new pod template, within the _openshift_ cloud previously configured, select *Add Pod Template* and then *Kubernetes Pod Template*
+
+Using the table below, create a new pod template for each Jenkins slaves that you would like to provide to users:
+
+[options="header"]
+|=========================================================
+|Name |Description
+
+|Name|Friendly name of the pod template
+
+|Label|Label that is applied by users to signify their intention to have their build execute this pod template. It is recommended that _maven_ or _nodejs_ be entered depending on the template  
+
+|=========================================================
+
+Next, define a container that will run within the pod by selecting *Add Container* -> *Container Template*
+
+Use the values in the following table to fill out the template
+
+[options="header"]
+|=========================================================
+|Name |Value
+
+|Name| *jnlp*
+
+|Image|_Image to use for the container_ (such as *registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7*) 
+
+|Working Directory| */tmp*
+
+|Arguments to pass to the command| *${computer.jnlpmac} ${computer.name}*
+
+|Allocate pseudo-TTY| *Unchecked*
+
+|Service Account| *jenkins* (This value is not mandatory and is dependent on the configuration of the target namespace)
+
+|=========================================================
+
+NOTE: Some of the above values may be hidden by default. Click the *Advanced* button either on the Pod Template or the cloud configuration to expose all available options
+
+Once all of the values have been defined, click *Save* to apply the changes.
+
+== Validating the Configuration
+
+This section describes the process to validate each of the above sections was completed successfully
+
+=== BuildConfig Synchronization
+
+To validate BuildConfig API objects are being synchronized properly, use one of the namespaces that were defined in the <<OpenShift Sync Plugin>> section and create a new BuildConfig within this namespace that uses the _JenkinsPipeline_ build type.
+
+The following is an example of a BuildConfig that can be used to test:
+
+----
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  name: sample-pipeline
+  labels:
+    name: sample-pipeline
+spec:
+  strategy:
+    type: JenkinsPipeline
```

Comment:
i am going to leave it as is as these guides typically are used by individuals ramping up on the OpenShift ecosystem. Rather be overly descriptive as possible